client/extensions/woocommerce/state/data-layer/orders/send-invoice/index.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { createNoteSuccess } from 'woocommerce/state/sites/orders/notes/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import {
+	orderInvoiceFailure,
+	orderInvoiceSuccess,
+} from 'woocommerce/state/sites/orders/send-invoice/actions';
+import request from 'woocommerce/state/sites/http-request';
+import { WOOCOMMERCE_ORDER_INVOICE_SEND } from 'woocommerce/state/action-types';
+
+export const fetch = action => {
+	const { siteId, orderId } = action;
+	return request( siteId, action ).post( `orders/${ orderId }/send_invoice`, {} );
```

Comment:
Minor, but I think that `{}` will get passed by default for post requests, so you can leave this off.